CryptoPrices/Sources/Home/Sources/Home/WalletStatisticSection.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  WalletStatisticSection..swift
+//  Home
+//
+//  Created by Minh Pham on 07/12/2022.
+//
+
+import Styleguide
+import SwiftUI
+
+public struct WalletStatisticSection: View {
+
+    public var body: some View {
+        ZStack {
+            gradientBackground
+            VStack(spacing: 40.0) {
+                totalCoins
+                todayProfit
+            }
+            .frame(
+                minWidth: 0,
+                maxWidth: .infinity,
+                alignment: .topLeading
+            )
+            .padding(EdgeInsets(top: 0.0, leading: 32.0, bottom: 0.0, trailing: 32.0))
+        }
+    }
+
+    var gradientBackground: some View {
+        Rectangle().fill(
+            LinearGradient(
+                gradient: Gradient(colors: [.metallicSeaweed, .tiffanyBlue]),
+                startPoint: .topLeading,
+                endPoint: .bottomTrailing
+            )
+        )
+        .cornerRadius(12.0)
+        .frame(height: 212.0)
+        .padding(EdgeInsets(top: 0.0, leading: 16.0, bottom: 0.0, trailing: 16.0))
+        .shadow(color: .tiffanyBlue, radius: 16.0, x: 0.0, y: 16.0)
```

Comment:
I think we don't have to explicitly fix the frame's height here (I guess since we use `ZStack` we have to deal with it)
May I suggest a change like this 🙏 
```swift
struct WalletStatisticSection: View {

    var body: some View {
        VStack(spacing: 40.0) {
            totalCoins
            todayProfit
        }
        .padding(.horizontal, 16.0)
        .padding(.vertical, 32.0)
        .background(gradientBackground) <- use `background` instead of `ZStack` and no need `.frame(minWidth:maxWidth)`
    }
}

extension WalletStatisticSection {

    private var gradientBackground: some View {
        // No need `Rectangle().fill`
        LinearGradient(
            gradient: Gradient(colors: [.metallicSeaweed, .tiffanyBlue]),
            startPoint: .topLeading,
            endPoint: .bottomTrailing
        )
        .cornerRadius(12.0)  <- no need `.frame(height: 212.0)`
        .shadow(color: .tiffanyBlue, radius: 16.0, x: 0.0, y: 16.0)
    }
    ...
}
```

Reply:
Nice suggestion, and you are right that the explicitly fix the frame's height is needed for the `ZStack` or we can set the padding for it too 👍 By the way, there was just a small issue that the `VStack`'s horizontal padding should be `16.0` and there should be `gradientBackground`'s horizontal padding of `16.0` to match with the expected design. I help update to reflect that in [23322e7](https://github.com/nimblehq/nimble-crypto-ios/pull/28/commits/23322e77dc39ebcc0b586fae0003f560c76e159c) 🚀